### DISCUSSION TITLE
[Votes ended on 2023-08-30] Ban CLAs for collections included in the Ansible package

### DISCUSSION THREAD

**Discussion by gotmax23** [ansible-community/community-topics#269] [Votes done]
Vote `+1` to merge https://github.com/ansible/ansible-documentation/pull/288 or `-1` to proceed with the status quo. Keep all discussion in https://github.com/ansible-community/community-topics/issues/221 or https://github.com/ansible/ansible-documentation/pull/288. This discussion is only for voting.

**Comment by gotmax23** [comment]
+1 to my own proposal :)

**Comment by felixfontein** [comment]
+1

**Comment by ayushnix** [comment]
+1

**Comment by mariolenz** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by searge** [comment]
+1

**Comment by arichtman** [comment]
+1

**Comment by miah-oss** [comment]
+1

**Comment by Omripresent** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by pezhore** [comment]
+1

**Comment by alecunsolo** [comment]
+1

**Comment by aveiga** [comment]
+1

**Comment by purpleidea** [comment]
+1, although this only has relevant teeth if the license has some copyleft in it, otherwise someone can still take ALv2 (Apache License v2) licensed code and make future versions proprietary. So a vote to favour LGPLv3+ licensing would be a welcome addition too! Cheers!

**Comment by ghost** [comment]
+1. No CLA on Ansible. If this **EVER** happens, I'll switch to Nix.

**Comment by robertdahlem** [comment]
+1

**Comment by blockofdynamite** [comment]
+1

**Comment by minecraftchest1** [comment]
+1

**Comment by teeks99** [comment]
+1

**Comment by DasFaultier** [comment]
+1 to ban CLAs for collections

**Comment by fitbeard** [comment]
+1

**Comment by ggiesen** [comment]
+1

**Comment by kmf** [comment]
+1

**Comment by wirelessben** [comment]
+1

**Comment by Zorlin** [comment]
+1

**Comment by ringods** [comment]
+1

**Comment by tonk** [comment]
+1

**Comment by bertvv** [comment]
+1

**Comment by richardfontana** [comment]
I assume my vote doesn't count, but ... +1 :)

**Comment by ngotzmann** [comment]
+1

**Comment by lcndsmr** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by anweshadas** [comment]
+ 1

**Reply by cybette** [reply]
@anweshadas you might want to remove the space between `+` and `1`, currently it displays as a bullet point

**Comment by resmo** [comment]
+1

**Comment by samccann** [comment]
+1

**Comment by jwbowen** [comment]
+1

**Comment by CoaxVex** [comment]
+1

**Comment by schurzi** [comment]
+1

**Comment by jamescassell** [comment]
+1
